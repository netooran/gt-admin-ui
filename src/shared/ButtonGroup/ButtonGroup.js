import styled from 'styled-components';
import PropTypes from 'prop-types';

const Wrapper = styled.div`
  display: flex;

  button {
    margin-right: 0.5rem;
  }
`;

const ButtonGroup = ({ children }) => {
  return <Wrapper>{children}</Wrapper>;
};

ButtonGroup.propTypes = {
  children: PropTypes.array.isRequired,
};

export default ButtonGroup;
