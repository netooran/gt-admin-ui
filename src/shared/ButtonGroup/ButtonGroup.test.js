import { render } from '@testing-library/react';
import ButtonGroup from './ButtonGroup';

test('renders button group', () => {
  const { container } = render(
    <ButtonGroup>
      <button>Click Me</button>
      <button>Click Me</button>
    </ButtonGroup>
  );
  expect(container).toMatchSnapshot();
});
