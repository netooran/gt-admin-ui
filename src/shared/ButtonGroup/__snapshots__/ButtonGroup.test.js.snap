// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button group 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c0 button {
  margin-right: 0.5rem;
}

<div>
  <div
    class="c0"
  >
    <button>
      Click Me
    </button>
    <button>
      Click Me
    </button>
  </div>
</div>
`;
